pubpub/pubpub
# feature: in-Pub collection browsing

This PR adds some UI that adds collection-level context to the Pub reading experience. We need this for Frankenbook, but I'm hoping that it'll be useful more generally.

In the header, we have a new dropdown that shows all the other pubs available in a pub's primary collection, along with arrow buttons leading to the previous and next pubs in the collection. If there's no primary collection, this UI won't be shown. I've also added a little overflow menu to the collection tags that prevent them from needing to scroll horizontally across the header.

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/2208769/60044957-8b1f0100-9691-11e9-87ca-71f135dbf8fa.png">

At the end of the pub, we'll also show a large `PubPreview` for the next pub in the primary collection, if it exists:

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/2208769/60044984-a12cc180-9691-11e9-8fb3-096a342cb7bd.png">

The links in these new elements add contain a `readCollection` URL parameter that takes precedence over the primary collection of a pub. That means that if we're on pub A and it has collection X as its primary, navigating to pub B will keep us in collection X even if pub B's primary is collection Y (hopefully that makes sense!)

_Test plan:_
- Visit a pub with a primary collection and make sure the primary collection browser works as expected. Try clicking through pubs in the dropdown menu and using the arrows to navigate from beginning to end.

- Check the bottom of these pubs (right below the footer) for the Read Next UI and verify that its links work.

- Verify that a pub with no primary collection loads as expected, but doesn't have either the collection browser or Read Next elements.
